Sources/Debug Plugin/HeroDebugView.swift

```diff
@@ -162,7 +162,6 @@ class HeroDebugView: UIView {
       startLocation = pinchGR.location(in: nil)
       startTranslation = translation
       startScale = scale
-      fallthrough
```

Comment:
Why did you remove `fallthrough` here? I think this needs to fall to the case below